app/dashboard/templates/bounty/new.html

```diff
@@ -169,6 +190,9 @@ <h5 class="font-subheader pb-2">{% trans "Gas Settings" %}</h4>
 <script src="{% static "v2/js/tokens.js" %}"></script>
 <script src="{% static "v2/js/shared.js" %}"></script>
 <script src="{% static "v2/js/pages/shared_bounty_mutation_estimate_gas.js" %}"></script>
+<script>
+    const GITCOIN_TIP_ADDRESS = "{{gitcoin_tip_address}}";
```

Comment:
Can you space this out to `"{{ var }}"` vs `"{{var}}"` ?

Reply:
Done